jerry-core/parser/regexp/re-parser.h

```diff
@@ -78,7 +78,7 @@ typedef enum
 /**
  * Undefined character (out of the range of the codeunit)
  */
-#define RE_CHAR_UNDEF 0xFFFFFFFF
+#define RE_CHAR_UNDEF 0xFFFF
```

Comment:
Because only the RegExp use it :) But I'm not against to move this define to `lit-strings.h` as `LIT_UNDEFINED_CODE_UNIT` or `LIT_UNDEF_CHAR`. But this value has already a define in lit-strings.h: `#define LIT_UTF16_CODE_UNIT_MAX (0xFFFF)`
If we remove `RE_CHAR_UNDEF` and use `LIT_UTF16_CODE_UNIT_MAX`, then the code of RegExp parser will be less readable.